tests/test_qnode.py

```diff
@@ -896,11 +893,12 @@ def circuit(x, y):
         ],
     )
     def test_defer_meas_if_mcm_unsupported(
-        self, dev, first_par, sec_par, return_type, mv_return, mv_res, mocker
+        self, dev_name, first_par, sec_par, return_type, mv_return, mv_res, mocker
     ):  # pylint: disable=too-many-arguments
         """Tests that the transform using the deferred measurement principle is
         applied if the device doesn't support mid-circuit measurements
         natively."""
+        dev = qml.device(dev_name, wires=3)
```

Comment:
Shouldn't the warning be getting filtered here?

Reply:
No cuz it would be captured by the fixture